src/olympia/devhub/tasks.py

```diff
@@ -369,6 +369,15 @@ def annotate_legacy_addon_restrictions(results, is_new_upload):
         insert_validation_message(
             results, message=msg, msg_id='thunderbird_and_seamonkey_migration')
 
+    # Legacy add-ons are no longer supported on AMO (if waffle is enabled).
+    elif (is_extension_or_complete_theme and
```

Comment:
Actually... I don't think there is any other type we'd want to accept.

Reply:
search plugins?  (dictionaries and language packs are webext now?)

Reply:
mm yeah search plugins would still need to be allowed. But legacy language packs or dictionaries shouldn't be accepted anymore with this waffle enabled IMHO.

Reply:
@jvillalobos the issue only specified extensions and complete themes - are other addon types to be blocked too?

Reply:
I think language packs is covered by https://github.com/mozilla/addons-linter/issues/1985. Legacy dictionaries are still allowed (didn't make it to ESR 60, per [bug 1410214](https://bugzilla.mozilla.org/show_bug.cgi?id=1410214)). Search plugins are still allowed for now.